webapp/tests/NoDBLock/Command/ResetUserPasswordCommandTest.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
+
+namespace App\Tests\Unit\Command;
+
+use App\Tests\Unit\BaseTest as BaseTest;
+use Symfony\Component\Process\Process;
+
+class ResetUserPasswordCommandTest extends BaseTest
+{
+
+    public static $demoPassword = 'demo';
+
+    public function testNonExistingUser(): void {
+        $username = 'NotAnUser';
+        $command  = ['webapp/bin/console','domjudge:reset-user-password',$username];
```

Comment:
Instead of creating a process, can’t we get the command from the container and run it that way? Might that also solve the db lock issue?

Reply:
If you think that would make the test similar I can try it. Would be nice if it solves the DB lock issue.

Reply:
Yeah it’s creating the command, an ArrayInput and some output and calling run. Should maybe fix the db lock issue, not sure

Reply:
This didn’t work?

Reply:
It did, I just forgot to remove this file.